<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlopeTool</class>
 <widget class="QDialog" name="SlopeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slope Tool</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="ressources.qrc">:/icons/ressources/image-filter-hdr.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Normal to the plane in local cartesian (Y pointing north, X east)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="x_label">
           <property name="text">
            <string>X=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="y_label">
           <property name="text">
            <string>Y=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="z_label">
           <property name="text">
            <string>Z=</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="ressources.qrc">:/icons/ressources/fault-aki-conv.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="plane_info_label">
       <property name="text">
        <string>Plane information in Aki convention : Strike = ?  , Dip = ?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="message_label">
       <property name="text">
        <string>message</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="start_btn">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_btn">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
